dist/s_include_guards

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Check if every .h file has an include guard of any style:
+# either "#ifndef/#define {FILE}_H" or "#pragma once".
+
+. `dirname -- ${BASH_SOURCE[0]}`/common_functions.sh
+cd_top
+check_fast_mode_flag
+
+for f in $(
+    find bench ext src test tools -name \*.h |
+    fgrep -v wtperf_opt_inline.h |
```

Comment:
Nitpick: `grep -F` is preferable to `fgrep`, apparently newer GNU grep has deprecated the `fgrep`/`egrep` aliases (see WT-12568).

Reply:
Done